external/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if(NOT MSVC)
+	add_compile_options(-w)
+endif()
+
```

Comment:
I think hiding all warning on anything that's not MSVC is a pretty bad idea

Reply:
Do you want me to disable warnings for all compilers or to not disable them at all? I looked at the MSVC docs and it supports `-w` too,

Reply:
I have enabled `-w` for all compilers because I assume you do not want to see warnings for external code.

Reply:
Ah, my bad, for some reason I was assuming this was applying for Vita3K as well. But indeed it only applies to external dependencies. Fine by me

Reply:
Is this code below still necessary with this new way to hide warnings?

https://github.com/Vita3K/Vita3K/blob/e70999adcdd0537e882f8d5afe54843bc770d8b2/CMakeLists.txt#L236-L245

Reply:
removed